### PR TITLE
Updates version of InfiniteScroll to 2.3.1

### PR DESCRIPTION
**Pull request by vishy93**
Upgrade the InfiniteScroll plugin used to latest stable 2.3.1

### CODE REVIEW

File: app/design/frontend/base/default/layout/strategery-infinitescroll.xml [hunk 1]
```diff
@@ -57,7 +57,11 @@
         </reference>
         <!-- fix for configurable swatches: -->
         <reference name="product_list.name.after">
-            <block type="core/template" name="product_list.infinitescroll" template="strategery/infinitescroll/swatches.phtml"/>
+            <block type="core/template" name="product_list.infinitescroll">
+	        <action method="setTemplate" ifconfig="configswatches/general/enabled">
+                        <template>strategery_infinitescroll_swatches.phtml</template>
+             </action>
+	        </block>
```
Comment (gsomoza) [inline app/design/frontend/base/default/layout/strategery-infinitescroll.xml]: Why are these changes necessary?

Comment (vishy93) [reply app/design/frontend/base/default/layout/strategery-infinitescroll.xml]: @gsomoza Hey, to stop the config swatches html being added when it is disabled. Not needed when disabled.

Comment (gsomoza) [reply app/design/frontend/base/default/layout/strategery-infinitescroll.xml]: Another question: the name & path of the template changed from `strategery/infinitescroll/swatches.phtml` to `strategery_infinitescroll_swatches.phtml`, but it doesn't seem like that should happen and it seems to me this shouldn't even work.. am I missing something there?

**Comment by vishy93** [comment]
@gsomoza @damian-pastorini I have reverted the template path change. Should be all good to merge.

File: app/design/frontend/base/default/layout/strategery-infinitescroll.xml (hunk 1 above)
Comment (vishy93) [reply app/design/frontend/base/default/layout/strategery-infinitescroll.xml]: @gsomoza  Iv'e reverted the change
